src/utils/sentry.rs

```diff
@@ -5,7 +5,7 @@ use futures01::Poll;
 
 use sentry::{Hub, Scope};
 
-// TODO: where the fuck is the warning about the missing Debug impl???
```

Comment:
🤣

Reply:
i didn't mean for this to get checked in but... genuine question though.  Isn't the compiler supposed to warn about this?

Reply:
Maybe check the rust repo if there is a report about this? I noticed that `missing_doc_code_examples` is basically broken and I couldn’t even get it to work a single time.